content/docs/architecture/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Architecture"
+---
+The Orchestrator architecture comprises several integral components, each contributing to the seamless execution and management of workflows. Illustrated below is a breakdown of these components:
+
```

Comment:
nit: I would suggest highlighting the keywords in the bullet list to enhance the readability.